### DISCUSSION TITLE
PCL怎么实现的自动安装optifine

### DISCUSSION THREAD

**Discussion by mc-124** [Meloong-Git/PCL#3276] [提问] [answered]
没研究明白，谁能给个PCL的源码位置？我找不到

**Comment by WForst-Breeze** [comment, marked as answer]
https://github.com/Hex-Dragon/PCL2/blob/8c8b1066bff6476a14437a14bc16a281e25145bd/Plain%20Craft%20Launcher%202/Pages/PageDownload/ModDownloadLib.vb#L573
由此行起往后
